src/components/RangeSlider/index.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Error from '../Error';
+
+import styles from './index.scss';
+
+const RangeSliderComponent = ({
+  id,
+  step,
+  min,
+  max,
+  name,
+  onBlur,
+  meta: {
+    touched,
+    error,
+    value,
+  },
+  onChange,
+  label,
+}) => (
+  <>
+    <div className={styles.sliderWrapper}>
+      <label htmlFor={id}>
+        {label}
```

Comment:
hm, we have a label component as well, maybe we should refactor this to actually make use of this

Reply:
There is no additional value is using it in this case.